### ISSUE TITLE
Problem with preview of the avatar.

### ISSUE THREAD

**Issue by AdamKasp** [Sylius/Sylius#10479]
When I'm adding Avatar to admin account I don't see a preview of my photo. Preview of the foto is seen only after I save changes. IMO photo should be visible after uploading, not after saving.

Next thing, is redirecting after saving changes. After that, we should stay on administrator show page instead of being redirected to the administrator index page.

**Comment by kulczy** [comment]
It would be nice to have an avatar removal button too

**Comment by CoderMaggie** [comment]
Fixed by #10510
